pkg/disttask/framework/scheduler/scheduler.go

```diff
@@ -727,6 +729,11 @@ func (s *BaseScheduler) WithNewTxn(ctx context.Context, fn func(se sessionctx.Co
 	return s.taskMgr.WithNewTxn(ctx, fn)
 }
 
+func (*BaseScheduler) isStepSucceed(cntByStates map[proto.SubtaskState]int64) bool {
```

Comment:
```suggestion
func isSucceed(cntByStates map[proto.SubtaskState]int64) bool {
```

No step input

Reply:
the input is subtask count map for current step

Reply:
> the input is subtask count map for current step

For the current function, we don't know this background🤔

Reply:
I know the input. But the input that make sense of `isStepSucceed` should be task——step of task is succeed.

Reply:
we're in task scheduler already, `isStepSucceed` is clear enough